homeassistant/components/synology_dsm/__init__.py

```diff
@@ -314,6 +351,7 @@ def signal_sensor_update(self) -> str:
 
     async def async_setup(self):
         """Start interacting with the NAS."""
+        # init SynologyDSM object and login
```

Comment:
This comment seems very obvious. Comments should not describe the code. They should explain why the code is written like it is. If there's no reason to question why the code does something, a comment isn't needed.